Explorer/Assets/DCL/EmotesWheel/EmoteWheelSlotView.cs

```diff
@@ -66,15 +63,19 @@ public void OnPointerEnter(PointerEventData eventData)
         {
             UIAudioEventsBus.Instance.SendPlayAudioEvent(HoverAudio);
             hoverBackground.SetActive(true);
-            SlotAnimator.SetTrigger(AnimationHashes.HOVER);
+            SlotAnimator.SetTrigger(UIAnimationHashes.HOVER);
             OnHover?.Invoke(Slot);
         }
 
         public void OnPointerExit(PointerEventData eventData)
         {
             hoverBackground.SetActive(false);
-            SlotAnimator.SetTrigger(AnimationHashes.UNHOVER);
+            SlotAnimator.SetTrigger(UIAnimationHashes.UNHOVER);
             OnFocusLeave?.Invoke(Slot);
         }
+
+        public event Action<int>? OnPlay;
+        public event Action<int>? OnHover;
+        public event Action<int>? OnFocusLeave;
```

Comment:
I think it was our code style reformat DCL thing... it moves it to the bottom... I will move it back, but if we touch this file again and reformat, it will go down again xD